demos/common/cpp/utils/include/utils/common.hpp

```diff
@@ -249,3 +250,69 @@ inline void logExecNetworkInfo(const InferenceEngine::ExecutableNetwork& execNet
         }
     }
 }
+
+inline
+void log_compiled_model_info(
+    const ov::runtime::CompiledModel& compiledModel,
+    const std::string& modelName,
+    const std::string& deviceName,
+    const std::string& modelType = "")
+{
+    slog::info << "The " << modelType << (modelType.empty() ? "" : " ") << "model " << modelName << " is loaded to " << deviceName << slog::endl;
+    std::set<std::string> devices;
+    for (const std::string& device : parseDevices(deviceName)) {
+        devices.insert(device);
+    }
+
+    if (devices.find("AUTO") == devices.end()) { // do not print info for AUTO device
+        for (const auto& device : devices) {
+            try {
+                slog::info << "\tDevice: " << device << slog::endl;
+                std::string nstreams = compiledModel.get_config(device + "_THROUGHPUT_STREAMS").as<std::string>();
+                slog::info << "\t\tNumber of streams: " << nstreams << slog::endl;
+                if (device == "CPU") {
+                    std::string nthreads = compiledModel.get_config("CPU_THREADS_NUM").as<std::string>();
+                    slog::info << "\t\tNumber of threads: " << (nthreads == "0" ? "AUTO" : nthreads) << slog::endl;
+                }
+            }
+            catch (const ov::Exception&) {}
+        }
+    }
+}
+
+inline
+void log_model_info(const std::shared_ptr<ov::Model>& model) {
```

Comment:
yes, I find it useful to dump basic model info just after we read it. for compiled models we usually dump only device configuration

Reply:
@fzhar, @ivikhrev, @anzhella-pankratova, I have perception, that the decision was not to have it. Is this true?

Reply:
Actually, I don't remember the reason, why it was not introduced along with Python implementation, but this is an useful option. 
 @vladimir-dudnik could you please align logging messages with [Python version](https://github.com/openvinotoolkit/open_model_zoo/blob/master/demos/common/python/openvino/model_zoo/model_api/models/model.py#L198)? The info per layer should be printed on the one line, not splitted into 3 lines.

Reply:
Initially we agreed only to this logging stages (https://github.com/openvinotoolkit/open_model_zoo/pull/2547):

> 1. IE version
> 2. Model loading completion (with name and device)
> 3. IE parameters (num of streams, threads, requests)
> 4. Perfomance metrics.

Argument against the stage with logging information about input/output layers - that this would be too excessive for demos. In python this stage was introduced after we done logging unification, and I don't remember discussions about it.